code/game/objects/items/reagent_containers/hypospray.dm

```diff
@@ -32,19 +32,10 @@
 	if(tgui_alert(user, "Are you sure you want to empty [src]?", "Flush [src]:", list("Yes", "No")) != "Yes")
 		return
 	if(isturf(user.loc))
-		to_chat(user, span_notice("You flush the contents of [src]."))
+		user.balloon_alert(user, "You flush the contents of [src].")
```

Comment:
```suggestion
		balloon_alert(user, "Flushes")
```
Try to keep balloon alerts as small as possible, try not to insert names like [src] unless absolutely necessary.

Reply:
breaks code